server/controllers/oneEvent.js

```diff
@@ -0,0 +1,33 @@
+const {
+  Event, Coffee, Lunch, Equipment,
+} = require('../database/models');
+
+exports.oneEvent = ((req, res, next) => {
+  const { id } = req.params;
+  Event.findOne(
+    { where: { id }, raw: true },
+  ).then((event) => {
```

Comment:
You have to handle the errors that may come up inside each ```then()``` result. For example ```if(! event) {//do someting} ```

Reply:
done :+1: